packages/airnode-protocol/contracts/utils/interfaces/ISelfServeRrpBeaconServerWhitelister.sol

```diff
@@ -0,0 +1,37 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.9;
+
+interface ISelfServeRrpBeaconServerWhitelister {
+    event SetBeaconIdWithExpirationTimestamp(
+        bytes32 indexed beaconId,
+        uint64 expirationTimestamp
+    );
+    event SetBeaconIdWithIndefiniteWhitelistStatus(
+        bytes32 indexed beaconId,
+        bool indefiniteWhitelistStatus
+    );
+    event WhitelistedReaderWithExpiration(
+        bytes32 indexed beaconId,
+        address indexed reader
+    );
+    event WhitelistedReaderIndefinitely(
+        bytes32 indexed beaconId,
+        address indexed reader
+    );
+
+    function setBeaconIdWithExpirationTimestamp(
+        bytes32 _beaconId,
+        uint64 _expirationTimestamp
+    ) external;
+
+    function setBeaconIdWithIndefiniteWhitelistStatus(
+        bytes32 _beaconId,
+        bool _indefiniteWhitelistStatus
+    ) external;
+
+    function whitelistReaderWithExpiration(bytes32 _beaconId, address _reader)
+        external;
+
+    function whitelistReaderIndefinitely(bytes32 _beaconId, address _reader)
+        external;
+}
```

Comment:
Should we also add the 2 public getters for `beaconIdToExpirationTimestamp` and `beaconIdToIndefiniteWhitelistStatus` mappings?

Reply:
don't public mappings generate getters automatically?

Reply:
they do but if a contract only wants to import the interface to call any of the getters then they wouldn't be available